src/make.ts

```diff
@@ -54,6 +59,20 @@ export async function mkdist(
     };
   });
 
+  // Read and normalise TypeScript compiler options for emitting declarations
+  options.typescript ||= {};
+  options.typescript.compilerOptions = normalizeCompilerOptions(
+    defu({ noEmit: false }, options.typescript.compilerOptions, {
+      allowJs: true,
+      declaration: true,
+      incremental: true,
+      skipLibCheck: true,
+      strictNullChecks: true,
```

Comment:
Is it required for the strict flag? (thinking to comment and enable in next major)

Reply:
(This is the current behaviour - I've just moved the location where we set the default.)